components/Bookmark/index.js

```diff
@@ -1,38 +1,24 @@
-import React, { useState } from "react";
+import React from "react";
```

Comment:
I think you can remove this line, it is not required anymore since v17 (or so) of react 🙂